File: Javascript/3-Basic-Actions/1-Wait-Implicit.js
const {Builder, By} = require('selenium-webdriver');

(async function(){
    const driver = new Builder().forBrowser('chrome').build();

    try {
        (await driver).manage().setTimeouts({implicit:30000});

        (await driver).get("https://starandread.com/store/namebook")

        let my_element = (await driver).findElement(By.id('select-product'));
        console.log(await my_element.getText());

    } finally {
        (await driver).quit();
    }

})();

File: Python/3-Basic-Actions/1-Wait-Implicit.py
from selenium import webdriver

driver = webdriver.Chrome()
driver.implicitly_wait(30)
try:
    driver.get('https://starandread.com/store/namebook')
    my_element = driver.find_element_by_id('select-product')
    print(my_element.text)
finally:
    driver.quit()
